package.json

```diff
@@ -36,21 +37,23 @@
     "n-ui-foundations": "github:financial-times/n-ui-foundations#nobower",
     "next-session-client": "^3.0.1",
     "o-buttons": "npm:@financial-times/o-buttons@^6.0.2",
+    "o-icons": "npm:@financial-times/o-icons@^6.3.0",
```

Comment:
**Question**; Is this correct? To move from 7 to 6.3 ?

Reply:
This was an automatic change, I didn't touch this... must be something related to dependencies